src/features/search/pages/modals/VenueModal/useVenueModal.native.test.tsx

```diff
@@ -91,7 +91,7 @@ describe('useVenueModal', () => {
     expect(dismissMyModal).toHaveBeenCalled()
```

Comment:
Le linter Github Actions dit vrai, je crois que ESLint a même dû te crier dessus
```suggestion
    expect(dismissMyModal).toHaveBeenCalledWith()
```
s'il n'y a aucun paramètre.

Reply:
Ouep mais je pensais que ça passerai !

Reply:
Done

Reply:
done